spec/index.html

```diff
@@ -2011,14 +2011,14 @@ <h2>Application Profiles</h2>
           <p>This section identifies two application profiles for operating with
             YAML-LD:</p>
           <ul>
-            <li>the <a>YAML-LD JSON profile</a>, and</li>
-            <li>the <a>YAML-LD extended profile</a>.</li>
+            <li>the <a>YAML-LD Basic profile</a>, and</li>
+            <li>the <a>YAML-LD Extended profile</a>.</li>
           </ul>
 
           <p>Application profiles allow publishers to use YAML-LD
             either for maximum interoperability,
             or for maximum expressivity.
-            The <a>YAML-LD JSON profile</a> provides for complete round-tripping
+            The <a>YAML-LD Basic profile</a> provides for complete round-tripping
             between YAML-LD documents and JSON-LD documents.
             The <a>YAML-LD extended profile</a> allows for
             fuller use of YAML features to enhance the ability to
```

Comment:
Just a better phrasing, imho
```suggestion
            broader use of YAML features to enhance the ability to
```